internal/http/middleware/auth.go

```diff
@@ -59,7 +59,7 @@ func MustNotBeAuthenticated(h http.Handler) http.Handler {
 
 			session.Save(r, w)
 
-			http.Redirect(w, r, "/apply", http.StatusSeeOther)
```

Comment:
What does tbe dashboard look like for people who haven't applied?

Reply:
If someone doesn't have an app yet their dash looks like this:
![Screenshot_2019-12-28 BoilerMake](https://user-images.githubusercontent.com/10781907/71546166-ffe53780-2961-11ea-83e2-f73c4d477f30.png)

And with that latest commit when someone signs up they're redirected to /apply, but after logging in they're redirected to /dashboard

Reply:
nice 👍